openaerostruct/functionals/total_aero_performance.py

```diff
@@ -17,11 +17,13 @@ def initialize(self):
 
     def setup(self):
         surfaces = self.options['surfaces']
-
-        self.add_subsystem('sum_areas',
-             SumAreas(surfaces=surfaces),
-             promotes_inputs=['*S_ref'],
-             promotes_outputs=['S_ref_total'])
+        user_supplied_S_ref = False
```

Comment:
If this is set as False here, then directly used in the logic, it's always the case. Is that intended behavior?

Reply:
Did you want this as an option which is passed in from the run script? I wasn't sure how to do that without cluttering up the whole thing.

Reply:
Right now the user would have to modify total_aero_performance.py to change what they're using? If that's the case, then yes, we do want it as an option.